text.pollinations.ai/server.js

```diff
@@ -308,6 +308,7 @@ export function getRequestData(req) {
     const model = data.model || 'openai';
     const systemPrompt = data.system ? data.system : null;
     const temperature = data.temperature ? parseFloat(data.temperature) : undefined;
+    const isPrivate = data.private === 'true' || data.private === true;
```

Comment:
maybe add a tolowercase? in case people come from python. or is that already happening somewhere? @mentatbot